src/styles/components/_header.scss

```diff
@@ -1,28 +1,16 @@
-/*
- * Copyright 2019 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-@import '../../../styles/settings/colors';
-@import '../../../styles/settings/global';
-@import '../../../styles/tools/breakpoints';
-@import '../../../styles/tools/mixins';
-
-// Disable when Web Components aren't loaded; the button does nothing.
-body.unresolved web-header .web-header__hamburger-btn {
-  visibility: hidden;
-}
+@import '../settings/colors';
```

Comment:
Do you think we should add back the license header to this and the other .scss files? My hunch is OSPO would like that.

Reply:
I agree but I can follow-up with a PR.